[dynamo] relax missing symbols runtime assert

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #121339



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

Differential Revision: [D54603361](https://our.internmc.facebook.com/intern/diff/D54603361)

IvanKobzarev committed:
[dynamo] relax missing symbols runtime assert

[ghstack-poisoned]

IvanKobzarev commented:
@IvanKobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54603361).

ezyang reviewed (approved):
can you xref the comment out with https://github.com/pytorch/pytorch/issues/119689

IvanKobzarev committed:
Update on "[dynamo] relax missing symbols runtime assert"




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang

Differential Revision: [D54603361](https://our.internmc.facebook.com/intern/diff/D54603361)

[ghstack-poisoned]

IvanKobzarev commented:
@IvanKobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D54603361).
